src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/DiaryAddAndEditEntryModel.swift

```diff
@@ -0,0 +1,65 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct DiaryAddAndEditEntryModel {
+
+	// MARK: - Init
+
+	init(_ type: DiaryEntryType) {
+		switch type {
+		case .contactPerson:
+			self.namePlaceholder = AppStrings.ContactDiary.AddEditEntry.person.placeholders.name
+			self.phonenumberPlaceholder = AppStrings.ContactDiary.AddEditEntry.person.placeholders.phonenumber
+			self.emailAddressPlaceholder = AppStrings.ContactDiary.AddEditEntry.person.placeholders.email
+
+		case .location:
+			self.namePlaceholder = AppStrings.ContactDiary.AddEditEntry.location.placeholders.name
+			self.phonenumberPlaceholder = AppStrings.ContactDiary.AddEditEntry.location.placeholders.phonenumber
+			self.emailAddressPlaceholder = AppStrings.ContactDiary.AddEditEntry.location.placeholders.email
+		}
```

Comment:
If this was extracted into something like `private func setPlaceholders(for entryType: EntryType)` you could reuse it in the other inits. Just a suggestion :)